src/core/MIGRATION.md

```diff
@@ -1173,6 +1173,7 @@ import { setup, start } from '../core_plugins/visualizations/public/legacy';
 | `ui/registry/field_formats`                       | `data.fieldFormats`                                          |                                                                                                                                      |
 | `ui/registry/feature_catalogue`                   | `home.featureCatalogue.register`                             | Must add `home` as a dependency in your kibana.json.                                                                                 |
 | `ui/registry/vis_types`                           | `visualizations.types`                                       | --                                                                                                                                   |
+| `ui/registry/doc_views`                           | `discover.addDocView`                                        | Currently shimmed in the old platform - use `import { setup } from '../kibana/public/discover'; setup.addDocView(...)`                    |
```

Comment:
is it ready to be consumed by new platform plugins? it means no webpack imports, no global angular

Reply:
No, it's still in the legacy world for now, good point. I will remove it.